dask_kubernetes/helm.py

```diff
@@ -0,0 +1,244 @@
+import asyncio
+import shutil
+import subprocess
+import warnings
+
+from distributed.deploy import Cluster
+from distributed.core import rpc
+from distributed.utils import Log, Logs, LoopRunner
+import kubernetes_asyncio as kubernetes
+
+from .auth import ClusterAuth
+from .core import _namespace_default
+
+if shutil.which("helm") is None:
+    raise ImportError(
+        "Missing dependency helm. "
+        "Please install helm following the instructions for your OS. "
+        "https://helm.sh/docs/intro/install/"
+    )
+
+
+class HelmCluster(Cluster):
+    """ Connect to a Dask cluster deployed via the Helm Chart.
+
+    This cluster manager connects to an existing Dask deployment that was
+    created by the Dask Helm Chart. Enabling you to perform basic cluster actions
+    such as scaling and log retrieval.
+
+    Parameters
+    ----------
+    release_name: str
+        Name of the helm release to connect to.
+    namespace: str (optional)
+        Namespace in which to launch the workers.
+        Defaults to current namespace if available or "default"
+    port_forward_cluster_ip: bool (optional)
+        If the chart uses ClusterIP type services, forward the ports locally.
+        If you are using ``HelmCluster`` from the Jupyter session that was installed
+        by the helm chart this should be ``False``. If you are running it locally it should
+        be ``True``.
```

Comment:
I don't think it's possible to query the status of a `helm` release if you're on a pod inside of that release, is it?

Reply:
If you copy your local `~/.kube/config` to the Jupyter pod you should be able to. Perhaps this should be documented?

Perhaps we should also consider adding a service role to the chart which allows modifying the worker deployment replica count and getting pod logs. Although that could have security implications.